src/SprykerEco/Zed/Computop/Business/Hook/Mapper/Init/AbstractMapper.php

```diff
@@ -51,7 +51,6 @@ public function updateComputopPaymentTransfer(
         QuoteTransfer $quoteTransfer,
         TransferInterface $computopPaymentTransfer
     ): TransferInterface {
-        /** @var \Generated\Shared\Transfer\ComputopDirectDebitPaymentTransfer $computopPaymentTransfer */
```

Comment:
why was it removed?
typehint for this argument is `TransferInterface`

Reply:
Because of `No assignment to $computopPaymentTransfer variable before or after the documentation comment.` error message